bin/alluxio-start.sh

```diff
@@ -230,6 +230,14 @@ start_logserver() {
 }
 
 start_master() {
+  if [[ `${LAUNCHER} ${BIN}/alluxio getConf alluxio.master.journal.type` == "EMBEDDED" ]]; then
+    JOURNAL_DIR=`${LAUNCHER} ${BIN}/alluxio getConf alluxio.master.journal.folder`
+    if [ -f "${JOURNAL_DIR}" ]; then
+      echo "Journal location ${JOURNAL_DIR} is a file not a directory. Please remove the file before retrying."
+    elif [ ! -e "${JOURNAL_DIR}" ]; then
+      ${LAUNCHER} ${BIN}/alluxio format
```

Comment:
Should we do this after the `-f` check to prevent double format?

Reply:
done